settings.yml

```diff
@@ -0,0 +1,4 @@
+example_setting:
```

Comment:
This file you don't need. It'll add a setting in the UI.

Reply:
Done